assets/dev/scss/direction/app-base.scss

```diff
@@ -0,0 +1,14 @@
+$direction: ltr;
```

Comment:
Add an RTL version like other files in this folder.

Reply:
Handled.